kotlin-inject-compiler-core/src/main/kotlin/me/tatarka/inject/compiler/InjectGenerator.kt

```diff
@@ -498,6 +499,7 @@ class InjectGenerator(provider: AstProvider, private val options: Options) :
         return find(key)
     }
 
+    @Suppress("ReturnCount")
```

Comment:
Not a fan of this check, I love early returns. Should be disabled.

Reply:
:+1: